text/0059-token-metadata-document.md

```diff
@@ -0,0 +1,504 @@
+* Allow the same `ticker_symbol` to appear, or, allowing two `ticker_symbols` which differ only in case, or the replacement of letters with similar numbers.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This proposal has a few drawbacks relative to [MCIP 40](https://github.com/mobilecoinfoundation/mcips/pull/0040):
+
+* MCIP 40 puts the token metadata on chain. This makes it possible to enforce protocol level rules around how the document can evolve. It also ensures the availability of the document.
+* It also would take longer to develop since it would require an enclave upgrade.
+
+This proposal's authentication scheme is a single ed25519 signature.
+
+* As an alternative, we could have a signature chain, such as an x509. This allows for more sophisticated strategies around the keeping the key secure.
+* As an alternative, we could have a k-of-n ed25519 multisignature. This would allow for the signing authority to be shared across several parties who can sign asynchronously.
+
+Since this signature can easily be performed off-line anyways, and the document isn't expected to change often, we think a single ed25519 signature is fine for this stop-gap proposal.
+
+In our design, we included SVG images inline in the metadata document to avoid this validation issue.
+This will not scale well to e.g. millions of tokens, and we expect to transition to on-chain metadata before that becomes a problem.
+
+This proposal does not envision having different token registries per network. It is believed that this complexity is unnecessary.
+However, it means that just as clients need to hit Intel's IAS services as part of testing (in prod and dev), they will have to hit
+the `mobilecoin.com` URLs as well, so this is another way in which testing a deployed network will not be completely isolated to the servers deployed as part of the network.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+We chose JSON instead of protobuf to make it easier for clients (which may be web browsers) to directly consume this document in e.g. Javascript.
+
+We chose to separate the signature from the document to avoid the complexity of storing serialized json within json.
+This also allows that we can modify the signature scheme in the future while maintaining backwards compatibility, as long as we continue to serve both the old signature and the new signature,
+at different URLs.
+
+We chose not to attempt to standardize the localization of any cryptocurrency names, since it does not seem conventional to do so in cryptocurrency wallets.
+
+We chose to allow numbers and some punctuation to be used in ticker symbols.
+
+This is based on looking at token lists on various cryptocurrency exchanges:
+
+* [Binance.US token list](https://support.binance.us/hc/en-us/articles/360049417674-List-of-Supported-Assets)
+* Kraken's exchange includes the following symbols, where suffixes such as `.S` suffix are used with staked tokens and similar.
+
+```
```

Comment:
Suggest removing the full list and grab a few to highlight like what was done for Coinbase.
This full list feels like a wall of text getting in the way of the more pertinent information.